src/main/java/ch/njol/skript/aliases/ItemType.java

```diff
@@ -1418,6 +1418,22 @@ public Material getMaterial() {
 		return data.getType();
 	}
 
+	/**
+	 * @return A random block material this ItemType represents. {@link Material#AIR} will be returned if no block
+	 * 			material exists.
```

Comment:
outdated
```suggestion
	 * @return A random block material this ItemType represents.
	 * @throws IllegalStateException If {@link #isBlock()} is false.
```
Additionally, the exception should say `ItemType#hasBlock` (not ItemData, though ItemData does have `isBlock`)